bw2io/backup.py

```diff
@@ -61,25 +71,38 @@ def backup_project_directory(project: str):
     if project not in projects:
         raise ValueError("Project {} does not exist".format(project))
 
+    dir_backup = dir_backup or os.path.expanduser("~")
```

Comment:
If the user provides a directory you need to check if it is a directory, and if it has `os.W_OK`.

Reply:
done